shim/shim.c

```diff
@@ -238,7 +241,11 @@ handle_signals(char *container_id, int outfd) {
 	while (read(signal_pipe_fd[0], &sig, sizeof(sig)) != -1) {
 		printf("Handling signal : %d on fd %d\n", sig, signal_pipe_fd[0]);
 		if (sig == SIGWINCH ) {
-			cmd_type = WINSIZE;
+			cmd = strdup("winsize");
```

Comment:
No need for the underlying strlen + allocation here just define 2 static variables (static const char[]) and assign cmd to the right one.